Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_test.py

```diff
@@ -915,3 +915,25 @@ def json(self):
     mocker.patch('Elasticsearch_v2.requests.get', return_value=ResponseMockObject())
     result = Elasticsearch_v2.get_mapping_fields_command()
     assert result == expected_result
+
+
+def test_search_command_with_query_dsl(mocker):
+    """
+    Given
+      - index to the search command
+
+    When
+    - executing the search command
+
+    Then
+     - make sure that the index is being taken from the command arguments and not from integration parameters
+    """
+    import Elasticsearch_v2
+    Elasticsearch_v2.FETCH_INDEX = 'index from parameter'
+    mocker.patch.object(
+        demisto, 'args', return_value={'index': 'index from arg', 'query_dsl': 'test', 'size': '5', 'page': '0'}
```

Comment:
id extrcact "index from arg" to an argument

Reply:
done :)